hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionMetrics.java

```diff
@@ -1,13 +1,17 @@
 package pl.allegro.tech.hermes.api;
 
+import java.util.List;
 import java.util.Objects;
 
 public class SubscriptionMetrics {
     private long delivered;
     private long discarded;
     private long inflight;
-    private String rate = "0.0";
+    private String timeoutsM1;
```

Comment:
I'd rather call it value essence, not impl detail. Like in timespan vs timespanMs

Btw, what documentation?